README.md

```diff
@@ -71,3 +71,9 @@ $(eval $(call BuildPackageGluon,ffXX-hello-world))
 
 After creating the package, open a pull-request to the community-repository.
 
+
+## If you have commit access (in addition to the [Contributing Guidelines](CONTRIBUTING.md))
```

Comment:
Headline is far to long. Should be somewhat restructured to be positioned like the commit style guidelines.